_data/blogs.yml

```diff
@@ -267,3 +267,8 @@
   title: Here I come Coding In
   img: haider2.jpg
   author_name: Haider Shafaq
+  
+- url: https://technoorja.wordpress.com/2018/01/01/technoorja-learns-to-code/
+  title: Technoorja Learns To Code
+  img: Oorja_modified.jpg
```

Comment:
Looks good but upload this image

Reply:
I have uploaded that image

Reply:
Where?

Reply:
I have uploaded that image.

Reply:
In the img/blog  folder

Reply:
@oorjadandotiya The image is not uploaded in this commit!

Reply:
@codedsun 
Just re-uploaded the image to this commit - https://github.com/oorjadandotiya/gci17.fossasia.org/commits/gh-pages

Reply:
Please confirm if this has completed successfully.

Reply:
Just uploaded the image. PFB the link. 
https://github.com/oorjadandotiya/gci17.fossasia.org/blob/gh-pages/img/blog/Oorja1.png
Hope this time it works.